### ISSUE TITLE
Facing difficulties with the project documentantion

### ISSUE THREAD

**Issue by economuteo** [UPB-FILS-MA/questions#8]
# Question

After a brief discussion with one of the representatives of the laboratory, Teodor Dicu, we clearly understood that the initial idea for our project was not enough and in order for the project to be considered good enough, we needed to implement a Kalman filter.  Now we are kind of confused and don't know where to start. We are scared to follow our own trajectory, searching all over the place for documentation, hoping to make all the components fit together and implement the Kalman filter. Could you please help us with a trajectory and some materials?

## Hardware used
LCD 1602 
Temperature sensor
Raspberry PI PICO HW

# Materials read
<!-- place here any links to materials that are relevant to the question.
https://www.youtube.com/watch?v=IFeCIbljreY&ab_channel=VisuallyExplained

**Comment by alexandruradovici** [comment]
@DTeodor-Alexaandru can you take this please?

**Comment by DTeodor-Alexaandru** [comment]
Check out this well-written article on the Kalman Filter:
https://www.intechopen.com/chapters/76884

**Comment by alexandruradovici** [comment]
@economuteo do you have any additional questions?

**Comment by economuteo** [comment]
> @economuteo do you have any additional questions?

Yes! I have just a single question for now. What software should we use for the schematic in order for Teodor or Irina to print it on the 3D printer?

**Comment by economuteo** [comment]
> > @economuteo do you have any additional questions?
> 
> Yes! I have just a single question for now. What software should we use for the schematic in order for Teodor or Irina to print it on the 3D printer?

We have already thought about how the model is going to look like. We just need the software.

**Comment by alexandruradovici** [comment]
Software that exports to STL files should be fine. I think we used [TinkerCad](https://www.tinkercad.com).

**Comment by economuteo** [comment]
> Software that exports to STL files should be fine. I think we used [TinkerCad](https://www.tinkercad.com).

Ok! Thank you very much!
